src/components/Settings/Export/Export.component.js

```diff
@@ -282,6 +296,44 @@ class Export extends React.Component {
                   </div>
                 </ListItemSecondaryAction>
               </ListItem>
+              <Divider />
+              <ListItem>
+                <ListItemText
+                  className="Export__ListItemText"
+                  primary={<FormattedMessage {...messages.properties} />}
+                  secondary={<FormattedMessage {...messages.propertiesSize} />}
+                />
+                <ListItemSecondaryAction>
+                  <div className="Export__SelectContainer">
+                    {this.state.loadingAll && (
+                      <CircularProgress
+                        size={25}
+                        className="Export__SelectContainer--spinner"
+                        thickness={7}
+                      />
+                    )}
+                    <FormControl
+                      className="Export__SelectContainer__Select"
+                      variant="standard"
+                    >
+                      <InputLabel id="export-all-select-label-size">
+                        {intl.formatMessage(messages.size)}
+                      </InputLabel>
+                      <Select
+                        labelId="export-all-select-label-size"
+                        id="export-all-select-size"
+                        autoWidth={false}
+                        value={this.state.exportAllBoardSize}
+                        onChange={this.handleSizeChange}
+                      >
+                        <MenuItem value="12">Small</MenuItem>
```

Comment:
Add ´Small´, ´Medium´ and ´Large´ words to the ´Export.messages.js´ to get the translations 

Use this values instead:
 - Small: 9
 - Medium: 12
 - Large: 16

Remember to update the default value on the state definition! Line 42